src/scripts/main.js

```diff
@@ -1,3 +1,271 @@
 'use strict';
 
-// write your code here
+const button = document.querySelector('.button');
+
+const gameMessageStart = document.querySelector('.message-start');
+const gameMessageLose = document.querySelector('.message-lose');
+const gameMessageWin = document.querySelector('.message-win');
+
+const gameScore = document.querySelector('.game-score');
+const cells = document.querySelectorAll('.field-cell');
+
+const fieldSize = 4;
+const winValue = 2048;
+
+let score = 0;
+let field = Array.from({ length: fieldSize }, () => Array(fieldSize).fill(0));
+// field[ [], ... , []] - створено масив масивів, в залежності від fieldSize
```

Comment:
it's not good practice to add a lot of explanation comments in JS. We have to add comments in not obvious cases, when there are very complicated logic and so on

The code in your case is self-descriptive enough (that's a good sign :+1: ), it's clear what it does w/o comment, so delete them, please